shared/css/sidebar.css

```diff
@@ -259,42 +259,57 @@ html[dir="rtl"] .text-chat-entry.received {
   margin-right: 10px;
 }
 
-.text-chat-entry > p {
-  position: relative;
-  z-index: 10;
+.text-chat-entry p {
```

Comment:
Apologies for the fly-by, but why are we dropping child selectors for the slower performance descendant selectors?

I thought our conventions were to pretty much always to use child selectors, and I can't see a reason to change here.